app/test_main.py

```diff
@@ -1,3 +1,42 @@
+import pytest
```

Comment:
fix  ordering imports related to PEP 8  https://peps.python.org/pep-0008/#imports

Reply:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

Why should i import local application before third party import?